chat-plugins/info.js

```diff
@@ -460,6 +288,15 @@ var commands = exports.commands = {
 		}
 		this.sendReply(buffer);
 	},
+	datahelp:
+		"/data [pokemon/item/move/ability] - Get details on this pokemon/item/move/ability/nature." + "\n" +
+		"!data [pokemon/item/move/ability] - Show everyone these details. Requires: + % @ & ~",
+	detailshelp:
+		"/details [pokemon] - Get additional details on this pokemon/item/move/ability/nature." + "\n" +
+		"!details [pokemon] - Show everyone these details. Requires: + % @ & ~",
+	dthelp:
```

Comment:
For instance, I like my current way of implementing `/away` much better than the old way of reading `cmd`.